docassemble/ALToolbox/ThreePartsDate.py

```diff
@@ -175,11 +175,11 @@
   */
   var $col = $('<div class="col col-month">');
   
-  let id = date_id + '_month';
-  // '_ignore' prevents the field from being submitted, avoiding a da error
-  let name =  '_ignore_' + id;
+  var id = '_ignore_' + date_id + '_month';
```

Comment:
Any reason you changed `let` to `var` here, but not elsewhere? IMO it's either we do need it for compatibility, or we don't.

Reply:
You're right, we should be able to use `let` now. It's just an old habit. I'll change it back 👍